.forge-snapshots/removeLiquidity.snap

```diff
@@ -1 +0,0 @@
-120637
```

Comment:
whys this gone?

Reply:
its just that someone didn't take it out when they merged something into main... 

we really should be cleaning snapshots on every merge but people dont delete the folder so they get leftover. 

i run `rm -rf ./forge-snapshots` and then rerun them and so it catches things like this

Reply:
looks like at some point this was probably renamed to "simple removeLiquidity"